src/ui/osx/TogglDesktop/TimeEntryListViewController.m

```diff
@@ -710,4 +714,21 @@ - (void)deselectAllTimeEntryNotification
 	[self.collectionView deselectAll:self];
 }
 
+- (void)windowDidBecomeKeyNotification:(NSNotification *)notification
+{
+	// Don't focus on Timer Bar if the Editor is presented
+	if (self.timeEntrypopover.isShown)
```

Comment:
So, I fixed @skel35  issue by https://github.com/toggl/toggldesktop/pull/3223/commits/2c306ce440458d5e09a952aef5d94b2f760e86c8.

Basically, the logic "Always focus on Timer Bar when the app become key", which we've introduced recently causes a lot of problem with Time Entry Editor. For instance, when we close the AutoComplete (is separated NSWindow), the app will become Key, and make a app focus on Timer Bar -> Causes lost Focus, and couldn't Tab to move to next controls.

Therefore, I change a logic a bit:
- Shouldn't focus on the Timer Bar if the TE Editor is presented -> It make more sense, because the user's editing the TE.
- Only Focus on Timer Bar if the Editor is not presented, and come from background.

By following this new logic. I resolve the bug.